### ISSUE TITLE
1.0.0 Bug: Cannot specify Image height/width

### ISSUE THREAD

**Issue by joncursi** [oblador/react-native-image-progress#43]
After upgrading to `1.0.0`, a bunch of my Image components are not being correctly sized. I am applying height and width via the `style` prop:

```jsx
import Image from 'react-native-image-progress';

<Image
  ...
  source={...}
  style={{
    height: 50,
    width: 200,
  }}
  ...
/>
```

However on `1.0.0`, these `height` and `width` style attributes are not being respected and the image is not being resized. The images are rendering at the source image's natural height and natural width.

**Comment by joncursi** [comment]
Example - a tab bar with an image icon:

Before:
![img_3914](https://user-images.githubusercontent.com/4974609/27816502-f65df6c6-6059-11e7-9e5a-8d4e01fd1b1a.PNG)

After:
![screen shot 2017-07-04 at 1 41 18 am](https://user-images.githubusercontent.com/4974609/27816504-fb681250-6059-11e7-8b6a-59de4a167788.png)

**Comment by oblador** [comment]
Hi, for local static images like these you don't really need a progress image right? But will try to work on a fix.

**Comment by joncursi** [comment]
Theoretically yes, but in practice it makes for a much better developer experience to be able to standardize on a single `Image` component, rather than requiring the developer to remember to import different `Image` components from different sources, depending on the particular use-case.

**Comment by joncursi** [comment]
Some images will use the `uri` network interface to point to external images, some will use `require()` to point to a local source. Regardless of the location of the image, I'd expect both to behave the same / produce the same result:

1. show a progress indicator while fetching the image (if applicable, i.e. for network requests that take time)
2. render the image at specified dimensions (obey the `style` prop)

**Comment by oblador** [comment]
This should be fixed in [1.0.1](https://github.com/oblador/react-native-image-progress/releases/tag/v1.0.1).

**Comment by joncursi** [comment]
Indeed! Thank you!
